src/v8/remove-passport.ts

```diff
@@ -0,0 +1,22 @@
+import { existsSync } from "fs";
+
+import { PackageJson } from "../util/package-json.util";
+
+export const stage = "before-install";
+
+export default async function removePassport() {
+    if (!existsSync("api/package.json")) {
+        return;
+    }
+
+    const packageJson = new PackageJson("api/package.json");
+    packageJson.updateDependency("@nestjs/jwt", "^10.2.0");
```

Comment:
This should probably be 

```suggestion
    packageJson.addDependency("@nestjs/jwt", "^10.2.0");
```

since it wasn't a dependency up until now.